pom.xml

```diff
@@ -24,25 +24,25 @@
         <dependency>
             <groupId>net.snowflake</groupId>
             <artifactId>snowflake-jdbc</artifactId>
-            <version>3.16.1</version>
+            <version>3.20.0</version>
         </dependency>
```

Comment:
Should use scope `compile`. No idea why I overlook this...

Reply:
It should be `runtime`.